tests/snippets/json_snippet.py

```diff
@@ -16,36 +16,36 @@ def round_trip_test(obj):
 assert '[]' == json.dumps([])
 assert '[1]' == json.dumps([1])
 assert '[[1]]' == json.dumps([[1]])
-# round_trip_test([1, "string", 1.0, True])
+round_trip_test([1, "string", 1.0, True])
 
 assert '[]' == json.dumps(())
 assert '[1]' == json.dumps((1,))
 assert '[[1]]' == json.dumps(((1,),))
 # tuples don't round-trip through json
-# assert [1, "string", 1.0, True] == json.loads(json.dumps((1, "string", 1.0, True)))
-
-# assert '{}' == json.dumps({})
-# # TODO: uncomment once dict comparison is implemented
-# # round_trip_test({'a': 'b'})
-
-# assert 1 == json.loads("1")
-# assert -1 == json.loads("-1")
-# assert 1.0 == json.loads("1.0")
-# # TODO: uncomment once negative floats are implemented
-# # assert -1.0 == json.loads("-1.0")
-# assert "str" == json.loads('"str"')
-# # TODO: Use "is" once implemented
-# assert True == json.loads('true')
-# assert False == json.loads('false')
-# # TODO: uncomment once None comparison is implemented
-# assert None == json.loads('null')
-# assert [] == json.loads('[]')
-# assert ['a'] == json.loads('["a"]')
-# assert [['a'], 'b'] == json.loads('[["a"], "b"]')
-
-# class String(str): pass
-
-# assert '"string"' == json.dumps(String("string"))
+assert [1, "string", 1.0, True] == json.loads(json.dumps((1, "string", 1.0, True)))
+
+assert '{}' == json.dumps({})
+# TODO: uncomment once dict comparison is implemented
+# round_trip_test({'a': 'b'})
+
+assert 1 == json.loads("1")
+assert -1 == json.loads("-1")
+assert 1.0 == json.loads("1.0")
+# TODO: uncomment once negative floats are implemented
+# assert -1.0 == json.loads("-1.0")
+assert "str" == json.loads('"str"')
+# TODO: Use "is" once implemented
```

Comment:
`is` is implemented. What may not be implemented is two bool instances true and false. This can be added to context, by creating two instances of bool.

Reply:
I've filed #132 so we don't lose track of this issue.

Other than that I think this is ready to merge?